assets/section-featured-product.css

```diff
@@ -8,6 +8,10 @@
   z-index: 0;
 }
 
+.featured-product .product__media-item iframe, .featured-product .product__media-item video {
+  z-index: 1;
+}
+
```

Comment:
So the purpose of the z-index: 0 isn't super clear. There's no real context on that line in the [original commit](https://github.com/Shopify/dawn/pull/1124/commits/56080b13c8080e62fd93eaacaf515bf7543555d9) 
Side note: This is why I want to use code comments more 🙂 

But my theory right now is that it was an attempt to prevent the media shadow from being hidden behind the featured product background, before we more globally implemented the `.isolate` z-index fix on sections. In such a case I'd think we can remove that rule altogether. Testing, of course, that media drop shadows don't disappear behind from this section, with and without secondary background enabled.

Regarding the effect on focus outlines.. The white line is actually more expected currently (see main product media). But I also think the z-index: 0 currently makes the shadow overlap the border which is not correct https://screenshot.click/31-21-4lkep-j2msq.mp4

Reply:
We also have the option to add `pointer-events: none` on the pseudo element to solve the issue. I'm not fully in this context any more, but off-hand I'm not sure why that isn't already implemented for the shadow elements.

Reply:
Oh true, `pointer-events` could be a nice alternative. 🤔

Reply:
Cool, I messed with it quite a bit on my end and feel pretty good that it doesn't break anything, but I'll give it a final look later

Reply:
After removing `z-index: 0` the issue with video has gone. Why do we still need `pointer-event`?

Reply:
It just seemed like a good thing to have on it to future proof things as we never need to click the shadows 🤔 So that in case anything similar ever happens it won't be a problem.

Reply:
Yeah I don't believe I'm aware of a case where this would be affecting anything currently, but I don't think it's wrong to include it. Theoretically could be added to the other global element types (cards and content containers) but I'd have to retest that.